website/server/libs/amazonPayments.js

```diff
@@ -98,6 +98,9 @@ api.checkout = async function checkout (options = {}) {
 
   if (gift) {
     if (gift.type === this.constants.GIFT_TYPE_GEMS) {
+      if (gift.gems.amount <= 0) {
+        throw new BadRequest(i18n.t('badAmountOfGemsToPurchase'));
```

Comment:
Actually, we are using this on the front end as well, so maybe we just keep it in group translations for now?